app/services/xcode_manager_service.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "../shared/logging_module"
+require_relative "./services"
+
+module FastlaneCI
+  # Manages Xcode installations
+  class XcodeManagerService
+    def installer
+      if @_installer.nil?
+        @_installer = XcodeInstall::Installer.new
```

Comment:
`@_installer ||= XcodeInstall::Installer.new`?

Reply:
We're already checking for `nil?` so this won't be necessary 👍

Reply:
I mean, that `nil?` check is unnecessary if you just `return @_installer ||= XcodeInstall::Installer.new`?